helpers/setup.sh

```diff
@@ -14,15 +14,20 @@
 # limitations under the License.
 
 
-PROJECT_ID="$1"
-ORG_ID="$(gcloud projects describe $PROJECT_ID --format="value(parent.id)")"
+PROJECT_ID="$(gcloud projects list --format="value(projectId)" | grep -Eo "$1")"
+
+if [[ $PROJECT_ID == "" ]];
+then
+    echo "ERROR The specified project wasn't found."
+    exit 1;
+fi
+
+ORG_ID="$(gcloud projects describe ${PROJECT_ID} --format="value(parent.id)")"
```

Comment:
What about nested projects? Parent might be a folder.

Reply:
Good point. I'll need to think on this.

Reply:
Hmmm maybe I overthought this one. My initial thinking was that structurally, folders are considered to be suborganizations. So you can have one parent org, and then multiple suborgs. I wonder if it makes sense to just do something like...

gcloud organizations list --format="value(ID)" 

to grab the parent organization? Otherwise, might need to rely on the user here, and then supply an additional check to validate the org exists.

Reply:
Opened this to address it later: #3